components/docs-chef-io/content/automate/ha-aws-deployment-prerequisites.md

```diff
@@ -0,0 +1,203 @@
+    weight = 13
++++
+
+{{< warning >}}
+{{% automate/ha-warn %}}
+{{< /warning >}}
+
+{{< warning >}}
+The below pre-requisites are according to our organizational standard. If you are using any specified version not mentioned here or a third party extensions or software you can reach out to the CAMs and our Customer Support Team.
+{{< /warning >}}
+
+Before installing Chef automate HA on AWS deployment, ensure you have taken a quick tour of this pre-requisite page.
+
+## Hardware Requirements
+
+{{< note >}} Use a [Hardware Calculator](/calculator/automate_ha_hardware_calculator.xlsx) to check how much hardware you will need for your use case. {{< /note >}}
+
+To give you an apt hardware configuration, we have some sample values based on the performance benchmarking tests. You can refer to the below table to populate things in the **Hardware Calculator** according to your requirement. The below table is just based on the tested **assumptions** and does not has any exact value.
+
+You can use the below assumptions in the calculator to drive in to your hardware requirement:
+
+| Assumption                            | Value | Unit     |
+|---------------------------------------|-------|----------|
+| Number of Nodes sending data          | 5000  |          |
+| Frequency of Compliance Scan          | 1     | Per Hour |
+| Frequency of Client runs (Infra runs) | 1     | Per Hour |
+| Frequency of Event Feed               | 1     | Per Hour |
+| Data Retention policy                 | 1     | Days     |
+| Compliance scan report size           | 400   | KB       |
+| Client Run (Infra run) size           | 300   | KB       |
+| Event Feed update size                | 2     | KB       |
+| No. of Shards in OpenSearch Index     | 2     |
+
+The machine requirements based on the above assumptions are listed below:
+
+| Instance          | Count | vCPU | RAM | Storage Size(/hab) | AWS Machine Type | Additional Space |
+|-------------------|-------|------|-----|------------------- |------------------|------------------|
+| Chef Automate     | 2     | 2    | 8   | 80 GB              | m5.large         |/tmp=5%  /root=20%|
+| Chef Infra Server | 2     | 2    | 8   | 80 GB              | m5.large         |/tmp=5%  /root=20%|
+| Postgresql DB     | 3     | 2    | 8   | 150 GB             | m5.large         |/tmp=5%  /root=20%|
+| Opensearch DB     | 3     | 2    | 8   | 58.9 GB            | m5.large         |/tmp=5%  /root=20%|
+| Bastion Machine   | 1     | 2    | 8   | 150 GB             | m5.large         |/tmp=5%  /root=20%|
+
+{{< note >}}
+
+- For **OpenSearch** and **PostgresSQL**, a minimum of three node clusters is required.
+- For production, OpenSearch volume size also depends on the number of nodes and frequency of Chef Infra Client runs and compliance scans.
+{{< /note >}}
+
+### Load Balancer
+
+LoadBalancers in AWS deployment are set up according to [Chef Automate HA Architecture](/automate/ha/).
```

Comment:
LoadBalancers will be provisioned automatically, customer does not need to create manually in this kind of deployment